ui/ozone/platform/wayland/wayland_keyboard.cc

```diff
@@ -47,7 +49,8 @@ WaylandKeyboard::WaylandKeyboard(wl_keyboard* keyboard,
 
   wl_keyboard_add_listener(obj_.get(), &listener, this);
 
-  // TODO(tonikitoo): Default auto-repeat to ON here?
+  auto_repeat_handler_.SetCallback(
```

Comment:
as said before. Can we move this to the initialization list and pass the callback by reference? And remove SetCallback method of course.